README.md

```diff
@@ -167,8 +167,9 @@ More info: https://github.com/harvard-lil/scoop
 
 Options:
   -v, --version                                          Display Scoop and Scoop CLI version.
-  -o, --output <string>                                  Output path. (default: "./archive.wacz")
+  -o, --output <string>                                  Output path. (default: "[directory]/archive.wacz")
```

Comment:
I thought `.` was clearer than `[directory]` (unless we get to set `directory` somewhere).

Reply:
That's fair!